certbot/certbot/achallenges.py

```diff
@@ -47,7 +47,7 @@ def __getattr__(self, name: str) -> Any:
 
 class KeyAuthorizationAnnotatedChallenge(AnnotatedChallenge):
     """Client annotated `KeyAuthorizationChallenge` challenge."""
-    __slots__ = ('challb', 'domain', 'account_key')
+    __slots__ = ('challb', 'domain', 'account_key') # pylint: disable=redefined-slots-in-subclass
```

Comment:
With this, removing the duplicate `challb` slot ends up causing a behavior change in `josepy`, in that this exception is thrown:

https://github.com/certbot/josepy/blob/8f6adab8725f02bad50915640164ace911f39dd7/src/josepy/util.py#L162-L166

I didn't really dig into how `__slots__` work, how inheritance is meant to apply to them and whether we are doing the wrong thing in `josepy` above. For now, ignoring this problem seems to be the most pragmatic thing to do.